Source/OxyPlot.SharpDX.WPF45/D3D11Image.cs

```diff
@@ -1,75 +1,90 @@
-﻿//Stolen from https://github.com/sharpdx/Toolkit/blob/master/Source/Toolkit/SharpDX.Toolkit.Game/Desktop/D3D11Image.cs
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="D3D11Image.cs" company="OxyPlot">
+//   Copyright (c) 2014 OxyPlot contributors
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
 
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-using System.Windows.Interop;
-using SharpDX.Direct3D9;
-using SharpDX.Direct3D11;
-using DXGIResource = SharpDX.DXGI.Resource;
-using Device = SharpDX.Direct3D9.Device;
-using System.Windows;
+// Stolen from https://github.com/sharpdx/Toolkit/blob/master/Source/Toolkit/SharpDX.Toolkit.Game/Desktop/D3D11Image.cs
```

Comment:
Doesn´t the license terms say that you should not remove the license text from the original source code? SharpDX is also licensed under an MIT license and should be ok to use.
Or should we add a reference to SharpDX.Toolkit.Game?